tests/Fixtures/TestBundle/Document/SecuredDummy.php

```diff
@@ -30,7 +30,8 @@
  *         "post"={"access_control"="has_role('ROLE_ADMIN')"}
  *     },
  *     itemOperations={
- *         "get"={"access_control"="has_role('ROLE_USER') and object.getOwner() == user"}
+ *         "get"={"access_control"="has_role('ROLE_USER') and object.getOwner() == user"},
+ *         "put"={"access_control"="has_role('ROLE_USER') and original_object.getOwner() == user"},
```

Comment:
May I suggest to test both?

```suggestion
 *         "put"={"access_control"="has_role('ROLE_USER') and object.getOwner() == user and  original_object.getOwner() == user"},
```

Reply:
Testing both isn't ok @dunglas as we are trying to make sure that only the actual owner can update the owner to become someone else. This someone shouldn't be be able to assign the object to himself

Reply:
Well I can check owner for the "get" operation if you prefer, they are the same in this case...